pages/osx/bclm.md

```diff
@@ -0,0 +1,20 @@
+# bclm
+
+> Set a custom chrage limit on MacBooks.
+> More information: <https://github.com/zackelia/bclme>.
+
+- Set the charge limit to 80% (battery charge level may be slightly lower than set value for intel machines):
+
+`sudo bclm write 77`
```

Comment:
```suggestion
`sudo bclm write {{77}}`
```

Why is it 77 when the description says 80%?

Reply:
It is in order to show that the battery level is not always exactly the same as set limit.
I thought it was useful to show, but it is maybe not strictly necessary ?

Reply:
> It is in order to show that the battery level is not always exactly the same as set limit. I thought it was useful to show, but it is maybe not strictly necessary ?

Could you explain that? If the description says that the command sets the limit to 80%, then it should do exactly what the description tells you it does. If you think more information is needed, feel free to add something to the description in a way that's not misleading.

Reply:
the doc of the utility states the following:
> For Intel machines, when writing values, macOS charges slightly beyond the set value (~3%). In order to display 80% when fully charged, it is recommended to set the BCLM value to 77%. When charging while system is shut down or sleeping, the charging can go beyond set value more than average 3%.
and
> For Apple silicon machines, only the values 80 and 100 are supported and firmware >= 13.0 is required.

are those things that sould be understanble from the tldr page, or do we consider that people are going to go through the link if it seems that something is off ?

Reply:
Sounds fine, thanks.